browser/ui/views/commands/default_accelerators_mac.mm

```diff
@@ -50,6 +50,13 @@ bool CanConvertToAcceleratorMapping(NSMenuItem* item) {
 }
 
 AcceleratorMapping ToAcceleratorMapping(NSMenuItem* item) {
+  if (@available(macos 12.0, *)) {
+    // We can't parse keyEquivalent into keycode properly when it's unicode
+    // character. So before starting parsing, disable l10n for a while.
+    // https://github.com/brave/brave-browser/issues/31770
+    item.allowsAutomaticKeyEquivalentLocalization = NO;
```

Comment:
According to the docs(https://developer.apple.com/documentation/appkit/nsmenuitem/3787554-allowsautomatickeyequivalentloca?language=objc), setting `NO` here disables `allowsAutomaticKeyEquivalentMirroring` also.
Don't we need to `allowsAutomaticKeyEquivalentMirroring` value reset also when we set `YES` to `allowsAutomaticKeyEquivalentLocalization`

Reply:
Good point! Changed to reset it together.